pipelines/deploy.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
IMHO it should remain `sh` instead of bash. The script should work with any POSIX-shell `sh` may point to.

Reply:
Normally I'd agree but `mapfile` is a bash-specific functionality. Of course I could make `mapfile` available within other shells but we still would have to ensure that bash and therewith `mapfile` is available.

Reply:
But the `mapfile` function is not really required here. After all we don't require a map for the parameter overrides but a simple list suffices.
So 
``` 
PARAMETER_OVERRIDES=$(jq -r '.[] | del(select(."ParameterKey" == "ServiceImageVersion")) | values | "\(.ParameterKey)=\(.ParameterValue)"' "$PARAM_FILE")
``` 
will work just the same.

Reply:
Nope, that doesn't work as some of the values may contain whitespaces. `mapfile` ensures that such values are escaped correctly. Tried a similar solution as yours which failed because of that (see https://github.com/linked-planet/ktorbase/pull/5/commits/5473fad077d4cd813a3e516895db7472a1e923a9).

Reply:
In that case add escaped quotes to the output strings
```
...=$(jq -r '.[] | del(select(."ParameterKey" == "ServiceImageVersion")) | values | "\"\(.ParameterKey)=\(.ParameterValue)\""' "$PARAM_FILE")
```

Reply:
Sorry, I updated my last comment as it wasn't 100% correct.

That solution won't work as documented here: https://github.com/aws/aws-cli/issues/3274#issuecomment-628209188
I took the `mapfile` approach from a later comment within the same issue: https://github.com/aws/aws-cli/issues/3274#issuecomment-656880461

Reply:
I'll try the solution mentioned here to get rid off `mapfile`: https://stackoverflow.com/questions/41475261/need-alternative-to-readarray-mapfile-for-script-on-older-version-of-bash

Stay tuned! XD

Reply:
If you consider my latest suggestion, I'm still convinced it will work!
```
PARAMETER_OVERRIDES=$(jq -r '.[] | del(select(."ParameterKey" == "ServiceImageVersion")) | values | "\"\(.ParameterKey)=\(.ParameterValue)\""' "$PARAM_FILE" | tr '\n' ' ')
``` will produce a string like
```
"ServiceCpu=1024" "ServiceDeregistrationDelaySeconds=10" "ServiceDeploymentMinimumPercent=100" "ServiceDeploymentMaximumPercent=200"...
```
Which is exactly equivalent to what was passed to `--parameter-overrides` before the changes in this PR...

This also works for values containing spaces:
```
..."Environment=test" "ExternalSubnets=subnet-xxxxxxxxxxxxxx, subnet-zzzzzzzzzzzzzzz"...
```

Reply:
I'll try that :+1:

Reply:
Changed parameter format as suggested by @p-louis, now it works! :+1: 
Removed `mapfile` and switched to `/bin/sh` in shebang.

Thx @p-louis !